internal/datastore/revisions/hlcrevision.go

```diff
@@ -115,22 +145,11 @@ var (
 )
 
 // HLCKeyFunc is used to convert a simple HLC for use in maps.
-func HLCKeyFunc(r HLCRevision) string {
-	return r.String()
+func HLCKeyFunc(r HLCRevision) [2]int64 {
```

Comment:
I don't think this is really needed anymore - the current `HLCRevision` type is fine as map key

Reply:
It is, but I like the indirection in case we ever decide to change it again

Reply:
Changed